package.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "quiz",
+    "description": "Quiz",
+    "version": "1.0.0",
+    "scripts": {
+        "build": "webpack --env.mode=prod --env.watch=false",
+        "start": "webpack --env.mode=dev --env.watch=true",
+        "zip": "cd ./output && bestzip ./ActionPackage.zip *",
+        "deploy": "npm run build && npm run zip && upload-action-package -z ./output/ActionPackage.zip -t ./output/ --ld ./output/ --lp quiz",
+        "deploy-debug": "npm run build && npm run zip && upload-action-package -z ./output/ActionPackage.zip -t ./output/ -l debug --ld ./output/ --lp quiz",
+        "update-teams-app": "npm run build && npm run zip && upload-action-package -z ./output/ActionPackage.zip --ld ./output/ --lp quiz",
+        "inner-loop": "create-inner-loop -p com.microsoft.sample.quiz.mike06072010 -d ./output",
```

Comment:
please replace hardcoded Id with <packageId>

Also please use below for reference: https://github.com/OfficeDev/microsoft-teams-apps-survey/blob/main/package.json